gem_city.rb

```diff
@@ -0,0 +1,49 @@
+# This class represents the town of GemCity
+# This is a town riddled with crime but we can find out how happy the town is
+class GemCity
+  attr_reader :population
+
+  def initialize
+    @people = {
+      thieves: 5,
+      officers: 1
```

Comment:
Although Rubocop doesn't complain about it, this class could be simplified a bunch by dropping this hash and using `@theives` & `@officers`.